geode-core/src/main/java/org/apache/geode/redis/internal/executor/sortedset/GeoRadiusParameters.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal.executor.sortedset;
+
+import java.util.List;
+
+import org.apache.geode.cache.Region;
+import org.apache.geode.redis.internal.ByteArrayWrapper;
+import org.apache.geode.redis.internal.Coder;
+import org.apache.geode.redis.internal.CoderException;
+import org.apache.geode.redis.internal.GeoCoder;
+import org.apache.geode.redis.internal.GeoCoord;
+import org.apache.geode.redis.internal.MemberNotFoundException;
+import org.apache.geode.redis.internal.RedisCommandParserException;
+
+public class GeoRadiusParameters {
```

Comment:
I would prefer if this class's members were immutable, since it gets parsed once and that makes it clearer to the interfacing code that this is just an immutable data object.

Reply:
Sure, that's fair